benchmarks/2018.json

```diff
@@ -5071,35 +5017,19 @@
     "benchmarkYear": 2016,
     "performanceYear": 2018,
     "submissionMethod": "cmsWebInterface",
+    "isToppedOut": false,
```

Comment:
do you want these changes

Reply:
Ah... nope, it looks like this was changed in https://github.com/CMSgov/qpp-measures-data/pull/227/files in the json, but not the csv, so when I reran build-benchmarks these changes were overwritten. I should've reviewed the original PR more closely!